src/components/login/Login.js

```diff
@@ -81,17 +95,22 @@ class Login extends React.Component {
    */
   async login() {
     try {
+        /*
       const requestBody = JSON.stringify({
         username: this.state.username,
-        name: this.state.name
+        password: this.state.password
       });
-      const response = await api.post('/users', requestBody);
+         */
+      const requestHeader = ArrayBuffer.from(`${this.state.username}, ${this.state.password}`, 'utf8').toString('base64');
```

Comment:
There is no `ArrayBuffer.from`, you should use `const requestHeader = window.btoa(Uint8Array.from(this.state.username + ":" + this.state.password))`

Reply:
Actually, this generates a base64 code that is not easily consumed. Can you instead define the following function:
```
function toBinary(string) {
  const codeUnits = new Uint16Array(string.length);
  for (let i = 0; i < codeUnits.length; i++) {
    codeUnits[i] = string.charCodeAt(i);
  }
  return String.fromCharCode(...new Uint8Array(codeUnits.buffer));
}
```
and then do `window.btoa(toBinary(this.state.username + ":" + this.state.password))`.